src/uniprotkb/components/entry/InteractionSection.tsx

```diff
@@ -77,15 +72,13 @@ const InteractionSection = ({ data, primaryAccession }: Props) => {
       {interactionComment?.[0] && (
         <>
           <h3 data-article-id="binary_interactions">Binary interactions</h3>
-          {interactionViewerElement.defined && (
-            <interactionViewerElement.name accession={primaryAccession} />
-          )}
+          <interaction-viewer accession={primaryAccession} />
```

Comment:
Makes sense to keep the chunk splitting and having lazy loading, but I don't think unsupported browsers would crash with the registration implemented by Lit (I haven't tried, so I might be wrong!)?

Reply:
True, I'm sure they do the proper checks in order to avoid that.

In order to test non-support I usually do `delete window.customElements` in `index.tsx` to see how the page behaves